### PR TITLE
DLS-6213 Misc HTML fixes

### PR DESCRIPTION
**Pull request by adamliptrot-oc**
See related UI tests PR https://github.com/hmrc/cgt-property-disposals-ui-tests/pull/191

### CODE REVIEW

File: app/uk/gov/hmrc/cgtpropertydisposalsfrontend/views/returns/ytdliability/upload_mandatory_evidence.scala.html [hunk 1]
```diff
@@ -105,7 +104,6 @@
           errorKey = None,
           labelAsHeading = false,
           labelClass = Some("heading-medium bold"),
-          autoComplete = AutoCompleteType.On,
```
Comment (rambabu-posa) [inline app/uk/gov/hmrc/cgtpropertydisposalsfrontend/views/returns/ytdliability/upload_mandatory_evidence.scala.html:108]: will this autocompletion work without this flag?

File: app/uk/gov/hmrc/cgtpropertydisposalsfrontend/views/components/unit_input_govuk.scala.html [hunk 2]
```diff
@@ -103,7 +103,6 @@ <h1 class="govuk-label-wrapper">
               id="@fieldId"
               name="@fieldId"
               type="text"
-              inputmode="decimal"
```
Comment (rambabu-posa) [inline app/uk/gov/hmrc/cgtpropertydisposalsfrontend/views/components/unit_input_govuk.scala.html:106]: will this input component work for decimal value without this attribute?

**Comment by rambabu-posa** [comment]
@adamliptrot-oc  I found many unit test suites are failing in Jenkins job because of these UI component changes.
If required, please let me know I'm happy to fix them for you.

File: app/uk/gov/hmrc/cgtpropertydisposalsfrontend/views/returns/ytdliability/upload_mandatory_evidence.scala.html (hunk 1 above)
Comment (adamliptrot-oc) [reply app/uk/gov/hmrc/cgtpropertydisposalsfrontend/views/returns/ytdliability/upload_mandatory_evidence.scala.html:108]: autocomplete doesn't work on file inputs, so it is invalid on this element

File: app/uk/gov/hmrc/cgtpropertydisposalsfrontend/views/components/unit_input_govuk.scala.html (hunk 2 above)
Comment (adamliptrot-oc) [reply app/uk/gov/hmrc/cgtpropertydisposalsfrontend/views/components/unit_input_govuk.scala.html:106]: `decimal` has issues with some devices, so it is best not to set an `inputmode` at all
See https://design-system.service.gov.uk/components/text-input/#asking-for-decimal-numbers

**Review by rambabu-posa — APPROVED** [review]

**Pull request merged**